README.md

```diff
@@ -1,37 +1,21 @@
 # PlaceCal
 
-## Introduction / Warning
+## Introduction
 
-PlaceCal is a large and very complicated app for collating organisation and event information from a large variety of sources. In the words of one developer: "I'm not sure where the interface between the app and the real world is here".
+PlaceCal is an online calendar which lists events and activities by and for members of local communities, curated around interests and locality.
```

Comment:
Is there any wording we can think of to say that it is also a directory? Or something like that.

Reply:
i'm in the process of commissioning sarah for some new hero copy so we can change it then!